tendrl/commons/__init__.py

```diff
@@ -314,7 +314,7 @@ def setup_common_objects(self):
                        {"message": "Setup TendrlContext for namespace."
                                    "%s" % self.ns_name})
             self.current_ns.tendrl_context = \
-                self.current_ns.objects.TendrlContext()
+                self.current_ns.objects.TendrlContext().load()
```

Comment:
So this makes sure we load the latest value from etcd. But how are we making sure, we raise alert only when `integration_id` is available?

Reply:
all storage nodes should have integration_id when it is detected, so integration_id in tendrl_context should be available always. 

I feel we are not overriding or modifying tendrl_context anywhere except this place.

what my point is integtation_id is always available nowhere it is modified except this place in tendrl.

Reply:
And @shtripat when we restart node-agent and gluster integration back to back if some time integration_id not present then leads to lot of problem

Reply:
ack

Reply:
So this load would resolve that issue of integration_id not present for sometime as well?

Reply:
@shtripat yes it will make sure always integration_id present in tendrl_context object